benchexec/resources.py

```diff
@@ -40,7 +40,7 @@
            'get_cpu_package_for_core',
            ]
 
-def get_cpu_cores_per_run(coreLimit, num_of_threads, my_cgroups, coreSet=None):
+def get_cpu_cores_per_run(coreLimit, num_of_threads, my_cgroups, nohyperthreading, coreSet=None):
```

Comment:
Usually it is better to have variables named in a "positive" way, e.g., `hyperthreading` or `use_hyperthreading` instead of `nohyperthreading` (with negated meaning, of course). This makes it easier to understand the code.

The command-line parameter is not affected by this, because here the name depends on what the default is.

Reply:
Changed variable name to `use_hyperthreading`